HW_4/task_3.py

```diff
@@ -0,0 +1,33 @@
+"""
+Задать натуральное число k.
+Сформируйте многочлен (полином) степени k со случайными коэффициентами из промежутка от 0 до 100, включительно.
+Многочлен вывести в консоль и записать в файл.
+
+Ввод: значение типа <int>
+Вывод: значение типа <str>, файл с одной строкой.
+
+Пример:
+2
+2x^2 + 4x + 5 = 0
+"""
+
+
+from random import randint
+
+k = int(input('Введите натуральное число k:'))
+
+max_num = 100
+
+ratio = [randint(0, max_num) for i in range(k)] + [randint(1, max_num)]
```

Comment:
Не совсем понятно, почему вы не делаете так
```suggestion
ratio = [randint(0, max_num) for i in range(k + 1)]
```
Любой из коэффициентов полинома может быть равен нулю.